nemo-keyboard/org/nemomobile/KeyboardLandscape.qml

```diff
@@ -43,19 +43,25 @@ Column {
     property bool inSymView
     property bool inSymView2
 
-    property variant row1:["q1€", "w2£", "e3$", "r4¥", "t5₹", "y6%", "u7<", "i8>", "o9[", "p0]"]
-    property variant row2: ["a*`", "s#^", "d+|", "f-_", "g=§", "h({", "j)}", "k?¿", "l!¡"]
-    property variant row3: ["z@«", "x~»", "c/\"", "v\\“", "b'”", "n;„", "m:&"]
-    property variant accents_row1: ["", "", "eèéêë", "", "tþ", "yý", "uûùúü", "iîïìí", "oöôòó", ""]
-    property variant accents_row2: ["aäàâáãå", "", "dð", "", "", "", "", "", ""]
-    property variant accents_row3: ["", "", "cç", "", "", "nñ", ""]
+    property variant row1: KLayouts.keyboards[currentKeyboardLayout]["row1"]
```

Comment:
KLayouts is not defined
Missing `import "layouts.js"  as KLayouts`
https://github.com/nemomobile-ux/plugins/pull/3/files#diff-7a0f9e5afcaa997e56f722d487e15352R37